### DISCUSSION TITLE
Question about Part 10 – Add a digital display

### DISCUSSION THREAD

**Discussion by AndreiGr** [ketilmo/balena-ads-b#57] [Q&A] [answered]
Hello there,

I just saw that there is a possibility to add a digital display to the Pi and have it show a feeder page.

My question is, does this work only with HDMI displays, or could it work with a display like the one below (connects to the rPI through the pins):
![288832770_468557138311950_354714514576018468_n](https://user-images.githubusercontent.com/10116768/175252636-09efb3ed-e963-4cb1-8f4d-beea8442f08a.jpg)

**Comment by ketilmo** [comment, marked as answer]
Hi @AndreiGr,

That should work, but I have not tested it myself on such a screen. The functionality uses the balena dash module. Check out this blog repo for more details: https://github.com/balenalabs/balena-dash

Let us know if you can get it to work. Good luck!

Kind regards,
Ketil
